### PR TITLE
Issues/expand merriweather support

### PR DESCRIPTION
**Pull request by aerych**
This PR adds more varieties of the Merriweather font and adds examples of the font to the Example project. 

Needs Review: @bummytime

### CODE REVIEW

File: WordPress-iOS-Shared/Core/WPFontManager.m [hunk 1]
```diff
@@ -3,174 +3,136 @@
 
 @implementation WPFontManager
 
-static NSString * const kBundle = @"WordPress-iOS-Shared.bundle";
+static NSString * const SharedBundle = @"WordPress-iOS-Shared.bundle";
```
Comment (bummytime) [inline WordPress-iOS-Shared/Core/WPFontManager.m]: :+1:

File: WordPress-iOS-Shared/Core/WPFontManager.m [hunk 2]
```diff
@@ -3,174 +3,136 @@
...
+{
+    NSString *resourceName = @"Merriweather-Regular";
+    NSString *fontName = @"Merriweather";
+    return [self fontNamed:fontName resourceName:resourceName fontType:FontTypeTTF size:size];
+}
+
+
+#pragma mark - Private Methods
+
++ (UIFont *)fontNamed:(NSString *)fontName resourceName:(NSString *)resourceName fontType:(NSString *)fontType size:(CGFloat)size
```
Comment (bummytime) [inline WordPress-iOS-Shared/Core/WPFontManager.m]: Nice refactor.

**Comment by bummytime** [comment]
Looks great @aerych! :shipit:

**Comment by aerych** [comment]
Thanks @bummytime!

**Pull request merged**